nominations/mads-hartmann.md

```diff
@@ -0,0 +1,40 @@
+# Mads Hartmann
+
+## Proposer
+
+No one 😅 I'd sure like if someone would nominate me.
+
+## Qualifications and Experience
+
+- Worked together with Mirco Dotta and Iulian Dragos back in 2012 on
+  [Scala Semantic Search][scala-semantic-search]. It never fully made it into
+  Scala IDE for Eclipse (I think) but it did give me experience working with the
+  Scala Presentation Compiler and an appreciation for the challenges that
+  entails. Btw here's an [amazing demo][yes-that-took-6-months-to-build] of it 😉
+
+- Wrote one of the [more popular][scala.tmbundle] Scala syntax highlighting
+  definitions way back when I was using TextMate. I believe most editors
+  indirectly use this definition for highlighting (don't quote me on that). I
+  don't think it is directly relevant to the work we'll be doing in the LSP-WG
+  but it has convinced me that you can only do such much with regular
+  expressions before everything becomes one big mess 😉
+
+## Affiliations
+
+None
```

Comment:
Just to confirm, even if you're not representing any organization, any affiliations (e.g. through employment) should be listed here, just for transparency.

Reply:
Oh, I'll add my employer then :)